hack/prom-metric-linter/custom_linter_rules.go

```diff
@@ -0,0 +1,117 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2023 Red Hat, Inc.
+ *
+ */
+
+package main
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus/testutil/promlint"
+	dto "github.com/prometheus/client_model/go"
+)
+
+// projectParameters defines the operator and the sub-operator names
+var projectParameters = map[string]string{
+	"operator":     "kubevirt",
```

Comment:
@enp0s3 Hi, Would it better to have these parameters as `operator**_name**` and `sub_operator**_name**`?

Reply:
@sradco Hi. Regarding the naming convention, I think `operator` and `sub_operator` would be enough.

Reply:
We did used name at the end since we pass it as a flag.